test/surface_albedo.jl

```diff
@@ -0,0 +1,201 @@
+            atol = eps(FT),
+        ),
+    )
+
+    # test set_surface_albedo!(Y, p, t, α_type::RegressionFunctionAlbedo)
+    config.parsed_args["rad"] = "clearsky"
+    config.parsed_args["variable_surface_albedo"] = "RegressionFunctionAlbedo"
+    simulation = ClimaAtmos.get_simulation(config)
+    (; u, p, t) = simulation.integrator
+
+    ClimaAtmos.set_surface_albedo!(u, p, t, p.atmos.surface_albedo)
+
+    # test that the albedo is initiated (not NaN) - values are checked below
+    @test !isnan(sum(p.radiation.radiation_model.direct_sw_surface_albedo))
+    @test !isnan(sum(p.radiation.radiation_model.diffuse_sw_surface_albedo))
+end
+
+@testset "values of RegressionFunctionAlbedo against Jin et al 2011" begin
+    FT = Float32
+
+    # DIRECT
+    # visually confirmed values from Jin et al 2011 for n = 1.34
+    visually_confirmed_1 = [
+        0.021184867354289875,
+        0.021201913590205082,
+        0.021352182042848326,
+        0.022005125881264326,
+        0.024073276530937993,
+        0.02969349372976277,
+        0.043840283503901206,
+        0.07849404157000653,
+        0.1650409847710366,
+        0.37858968601087767,
+        0.7533128546339267,
+    ]
+    visually_confirmed_2 = [
+        0.021258497636095067,
+        0.021284274910078164,
+        0.02146584566098747,
+        0.022190639014997647,
+        0.02440628569506843,
+        0.030278402312514944,
+        0.04460644770841422,
+        0.07795363750458963,
+        0.15387494563952747,
+        0.3130321281076427,
+        0.5147225442522043,
+    ]
+    visually_confirmed_3 = [
+        0.021515607125057706,
+        0.021568549028609475,
+        0.021844851024032687,
+        0.02277521129128255,
+        0.025376772832975958,
+        0.03181634436201161,
+        0.04633667894977999,
+        0.0765734586254003,
+        0.1349348882833189,
+        0.2313406520807579,
+        0.32091070958565293,
+    ]
+
+    n_list = [1.34, 1.2]
+    θ_list = collect(0:(π / 20):(π / 2))
+    u_list = [0, 3, 12]
+    # plot = Plots.plot()
+    α_dir_list_all = []
+    for n in n_list
+        for wind in u_list
+            α_dir_list = []
+            for θ_rad in θ_list
+                albedo_model = ClimaAtmos.RegressionFunctionAlbedo{FT}(n = n)
+                push!(
+                    α_dir_list,
+                    ClimaAtmos.surface_albedo_direct(albedo_model)(
+                        FT(0),
+                        cos(θ_rad),
+                        FT(wind),
+                    ),
+                )
+            end
+            # plot!(
+            #     θ_list * 180 / π,
+            #     α_dir_list,
+            #     label = "n = $n, wind = $wind",
+            #     xlabel = "θ (deg)",
+            #     ylabel = "α_dir",
+            #     yaxis = :log,
+            #     yticks = ([0.001, 0.01, 0.1, 1], ["0.001", "0.01", "0.1", "1"]),
+            #     ylims = (0.001, 1),
+            #     legend = :topleft,
+            # )
```

Comment:
(Same with other comments.)

Reply:
I was hoping to leaving it here, in case someone wants to replot the plots that compare this with the paper plots. I can uncomment it, but in the other repos, we're trying not to depend on Plots in `/test`